knexfile.js

```diff
@@ -23,7 +23,8 @@ module.exports = {
   },
   production: {
     client: 'pg',
-    connection: 'postgres://mztouxgmsntjsh:5908998545484fb1a334ac1123311ecbded76f60212e6fe658de5fa154e3ce4c@ec2-174-129-255-76.compute-1.amazonaws.com:5432/d29t0b8jgphgn',
+    connection: `${process.env.PRODUCTION_DB}`,
+    // connection: 'postgres://mztouxgmsntjsh:5908998545484fb1a334ac1123311ecbded76f60212e6fe658de5fa154e3ce4c@ec2-174-129-255-76.compute-1.amazonaws.com:5432/d29t0b8jgphgn',
```

Comment:
Thanks for the insight into using environment variables here for security.  I added these in my local branch as well.